manual/docker/Enterprise-Edition/Deploy SeaTable-EE with Docker.md

```diff
@@ -2,7 +2,7 @@
 
 ## Requirements
 
-SeaTable Enterprise Edition (SeaTable EE) requires 4 cores and 8GB RAM. These resources guarantee good performance for most applications with several hundred concurrent connections.  When the bases become large, more RAM may be needed since SeaTable stores the bases in memory.
+SeaTable Enterprise Edition (SeaTable EE) requires 4 cores and 8GB RAM. These resources guarantee good performance for most applications with several hundred concurrent connections. When the bases become large, more RAM may be needed since SeaTable stores open bases in memory.
```

Comment:
if it is a requirement, then since which version? otherwise it's probably a recommendation (the question would be for which version still thought).

Reply:
I would say "stores" here is a verb, in German "speichert" - "SeaTable speichert Bases im Arbeitsgedächtnis". So the change is not necessary.

Reply:
Requirements apply to all versions.

Reply:
It's a requirement for all versions.